mne/epochs.py

```diff
@@ -308,8 +316,29 @@ def __init__(self, info, data, events, event_id=None, tmin=-0.2, tmax=0.5,
             else:
                 self.drop_log = drop_log
             events = events[selected]
-            if len(np.unique(events[:, 0])) != len(events):
-                raise RuntimeError('Event time samples were not unique')
+            unique_events, counts = np.unique(events[:, 0], return_counts=True)
+            if len(unique_events) != len(events):
+                if event_repeated == 'error':
+                    raise RuntimeError('Event time samples were not unique')
+                elif event_repeated == 'merge':
```

Comment:
I can think of another alternative. Instead of merging, you could drop the second event. What is the reason for preferring one over the other?

Reply:
I agree that this is an alternative, perhaps we can accept `"error"`, `"merge"`, and `"drop"`?